docs/nodes/configuration.md

```diff
@@ -409,16 +471,22 @@ peer-query-maj23-sleep-duration = "2s"
 #######################################################
 [tx-index]
 
-# What indexer to use for transactions
+# The backend database list to back the indexer.
+# If list contains null, meaning no indexer service will be used.
```

Comment:
Pardon my ignorance, does 'null' here mean the string `"null"`?

Reply:
it is either, a user can leave it blank or put in "null". I had to check this as well, ill add an excerpt about this to not confuse future people.